applications/main/nfc/plugins/supported_cards/troika.c

```diff
@@ -76,6 +90,132 @@ static bool troika_get_card_config(TroikaCardConfig* config, MfClassicType type)
     return success;
 }
 
+static TroikaLayout troika_get_layout(const MfClassicData* data, uint8_t start_block_num) {
+    furi_assert(data);
+
+    // Layout is stored in byte 6 of block, length 4 bits (bits 52 - 55), second nibble.
+    const uint8_t* layout_ptr = &data->block[start_block_num].data[6];
+    const uint8_t layout = (*layout_ptr & 0x0F);
+
+    TroikaLayout result = TroikaLayoutUnknown;
+    switch(layout) {
+    case 0x2:
```

Comment:
pointless - enum values are equal to switch cases, we can use them as-is.
So, just check for known values and cast to enum, return Unknown otherwise